src/lib/seam/components/SupportedDeviceTable/SupportedDeviceTable.tsx

```diff
@@ -8,11 +8,13 @@ import type { DeviceModelFilters } from 'lib/seam/components/SupportedDeviceTabl
 export interface SupportedDeviceTableProps {
   cannotFilter?: boolean
   className?: string
+  brands?: string[]
 }
 
 export function SupportedDeviceTable({
   cannotFilter = false,
   className,
+  brands = [],
```

Comment:
> Can we add the `brands` prop that will allow a user to filter for a subset of the brands? i.e. `<SupportedDeviceTable brands={ ["yale", "schlage"] } />` will return a table that only shows those two brands


Updated this in https://github.com/seamapi/react/pull/251/commits/ca2e488ff960df03cd2665ae68e8167152bf04d4

Also updated the filters to show only relevant brands if the user has passed them in.